packages/nx-stylelint/src/executors/lint/schema.json

```diff
@@ -45,6 +45,11 @@
       "type": "boolean",
       "description": "Hide output text.",
       "default": false
+    },
+    "allowEmptyInput": {
+      "type": "boolean",
+      "description": "Allow no results from the file patterns to exit without an error.",
```

Comment:
Please change the description to: "The executor exits without throwing an error when 'lintFilePatterns' match no files."

Reply:
Please move the option to the top of the properties object so that the properties are alphabetically sorted. 

Please also add the executor option to the `README.md`